cmd/evm/blockrunner.go

```diff
@@ -23,12 +23,12 @@ import (
 	"os"
 	"regexp"
 	"slices"
-	"sort"
 
 	"github.com/ethereum/go-ethereum/core"
 	"github.com/ethereum/go-ethereum/core/rawdb"
 	"github.com/ethereum/go-ethereum/tests"
 	"github.com/urfave/cli/v2"
+	"golang.org/x/exp/maps"
```

Comment:
use regular `maps` over `x/exp`

Reply:
That's why I had to use `keys := slices.Collect(maps.Keys(tests))`, because they changed maps.Keys to return an iterator when they moved from `x/exp` to native `maps`.

Reply:
Ah wait Collect was introduced in `go1.23.0 `. Nevermind, leave it as you have it now, I guess